hack/update-etcd-objects.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
Can we locate this file under cluster/ instead of hack/? I think that will be more reassuring to people actually using it on their cluster. Also, let's call it "update-storage-live.sh" or something like that.